File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int j = 1; j < nums.length; j++) {
            for (int i = 0; i < nums.length-j; i++) {
                if (nums[i]+nums[i+j] == target) return new int[] {i,i+j};
            }
        } return null;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen={}
        for i in range(len(nums)):
            result=target-nums[i]
            if result in seen:
                return [seen[result],i]
            seen[nums[i]]=i